main.go

```
@@ -87,6 +88,24 @@ var Db *badger.DB
 var Queues []Queue
 var DeadLetterQueue []Message
 
+var messageKeyCache sync.Map // key: "queueID\x00messageID" -> value: []byte (message key)
+
+func cacheMessageKey(queueID, messageID string, key []byte) {
+	messageKeyCache.Store(queueID+"\x00"+messageID, key)
+}
+
+func getCachedMessageKey(queueID, messageID string) ([]byte, bool) {
```

golangci-lint (2.11.4)

[error] func getCachedMessageKey is unused

(unused)